specification/cognitiveservices/AnomalyDetector/tspconfig.yaml

```diff
@@ -9,6 +9,7 @@ parameters:
     default: "{project-root}/azure-sdk-for-net/"
   "service-directory-name":
     default: "anomalydetector"
+imports: ["./client.tsp"]
```

Comment:
is that what we wanted to do?

I thought we wanted to keep the status-quo for now but just check that there wasn't issue with existing client.tsp impacting the service

Reply:
my opinion with doing this it that this is the same result as `import ./client.tsp` from the `main.tsp` but more hidden so this doesn't feel worth it.

```
service.tsp
client.tsp (import ./service.tsp;)
main.tsp (import ./service.tsp; import client.tsp)
```

think we could have structure otherwise and we can make sure you can compile all 3 potential entrypoint

Reply:
This is just loading client.tsp alongside main.tsp, which seems like the right thing when we are generating openapi

Reply:
Also, it makes tsp compile . work in all of these directories, which I like

Reply:
well yeah but that was the point of what I was suggesting to start with main.tsp should import client.tsp instead of the other way around so there is a single entrypoint. Here we are getting essentially the same thing but are getting into this issue where there might be things they have not designed/applied well which affect the service.

Reply:
Yeah, it is essentially the same, and for all the configured emitters, we actually want client.tsp.